tests/appsec/test_endpoint_ognl.py

```diff
@@ -0,0 +1,14 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, interfaces, context, irrelevant
+
+
+@irrelevant(context.library != "java", reason="Need to build endpoint on weblog")
```

Comment:
Sorry, I missed that : 


```suggestion
@missing_feature(context.library != "java", reason="Need to build endpoint on weblog")
```

Same for other endpoints checks. Except that, the rest is good for me

Reply:
Done